CHANGELOG.rst

```diff
@@ -44,7 +44,9 @@
 
 * Updated docstrings with a more uniform style.
 
-*
+* Testcase for overriding autouse fixtures with a parametrized one.
+  Thanks to `@astraw38`_ for reporting the issue (`#1601`_)
+  Thanks to `@diegorusso`_ for the PR
```

Comment:
IMHO there's no point in mentioning "internal" things like this in the changelog :wink: